proposals/spark-operator-adoption.md

```diff
@@ -0,0 +1,156 @@
+and simplifies lifecycle management of Spark Jobs on Kubernetes.
+
+This project boasts a substantial user base with 167 contributors, 2.5k likes, and is endorsed by 45
+organizations as documented in
+[the "Who is using" document](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/who-is-using.md).
+There are many more organizations using it than the originally reported 45. The primary issue at
+hand is that this project currently resides under the GoogleCloudPlatform GitHub organization, and
+it is solely moderated by a Google employee. Numerous users and customers have voiced concerns and
+raised issues regarding the maintenance of the repository. The existing Google maintainers
+are constrained by limitations in terms of time and support, adversely affecting both the project
+and its user community.
+
+At Kubecon Chicago 2023, AWS open source architects and the Apple infrastructure team initiated
+discussions with the Google team, specifically Marcin Wielgus. They conveyed their interest in
+contributing the project to either the Kubeflow or Apache Spark community.
+Marcin from Google confirmed their willingness to donate the project to either of these communities.
+
+Spark already has a lot of AI/ML use-cases and the Kubeflow ecosystem can help to address those.
+According to the recent Kubeflow survey, Spark is one of the most popular frameworks for Kubeflow users:
+
+![Kubeflow Survey 2022](diagrams/kubeflow-survey-2022.png)
+
+## Benefits for Kubeflow
+
+The following diagram shows the main components of Kubeflow. Notebooks for interactive ML development,
+Training Operator for distributed ML Training, Katib for HyperParameter Tuning, KServe for
+Model Serving, Pipelines for ML pipelines, and Profiles to create Kubeflow user profiles.
+
+![Kubeflow Overview](diagrams/kubeflow-overview.png)
+
+Today, Kubeflow doesn’t have any component for Data Preparation which is an essential step for
+MLOps lifecycle. Spark is one of the best and most-used frameworks for Data Preparation,
+Data Analysis, Feature Engineering for Data Scientists. Thus, adopting Spark Operator to Kubeflow
+components will significantly improve Kubeflow user experience.
+
+### Kubeflow Working Group Data
+
+Currently, Kubeflow doesn’t have a working group (WG) which is responsible for Data Preparation.
+By adopting Spark Operator to Kubeflow we can establish a new Data WG. The scope of this
+WG is to provide guidelines for AI/ML users to run data preparation tools such as
+[Spark](https://spark.apache.org/), [Dask](https://www.dask.org/), or [Ray](https://docs.ray.io/en/latest/)
+on Kubernetes. Also,
+[a new proposal for Model Registry](https://docs.google.com/document/d/1T3KfOqIfJohp0s1koQ2XrJJQQhj7TECO-m2xPsW59_c/edit#heading=h.t7b42oqmiz2y)
+in Kubeflow can be part of Data WG scope.
+
+Initially, Spark Operator is the ideal candidate to start this WG.
```

Comment:
We should note that this will be a separate proposal

Reply:
Sure, I added it @terrytangyuan!